client/app/bundles/course/assessment/submission/components/GetHelpChatPage/index.tsx

```diff
@@ -56,8 +76,14 @@ const GetHelpChatPage: FC<GetHelpChatPageProps> = (props) => {
       </div>
 
       <div className="relative flex flex-row items-center">
-        {isRenderingSuggestionChips && <SuggestionChips answerId={answerId} />}
-        <ChatInputArea answerId={answerId} questionId={questionId} />
+        {isRenderingSuggestionChips && (
+          <SuggestionChips answerId={answerId} syncStatus={syncStatus} />
+        )}
+        <ChatInputArea
+          answerId={answerId}
+          questionId={questionId}
+          syncStatus={syncStatus}
+        />
```

Comment:
`ChatInputArea` is disabled when thread is expired, but `SuggestionChips` are still clickable, which dispatches the message?

Reply:
disabled also for SuggestionChips